packages/core/handlers/routers/auth.js

```diff
@@ -1,26 +1,36 @@
 const { createIntegrationRouter } = require('@friggframework/core');
 const { createAppHandler } = require('./../app-handler-helpers');
-const { requireLoggedInUser } = require('./middleware/requireLoggedInUser');
 const {
-    moduleFactory,
-    integrationFactory,
-    IntegrationHelper,
+    loadAppDefinition,
 } = require('./../backend-utils');
+const { UserRepository } = require('../../user/user-repository');
+const { IntegrationFactory, IntegrationHelper } = require('../../integrations/integration-factory');
+const { GetUserFromBearerToken } = require('../../user/use-cases/get-user-from-bearer-token');
+
+const { integrations, userConfig } = loadAppDefinition();
+const userFactory = new UserFactory(userConfig);
+const integrationFactory = new IntegrationFactory(integrations);
+const userRepository = new UserRepository({ userFactory });
+const getUserFromBearerToken = new GetUserFromBearerToken({ userRepository });
 
 const router = createIntegrationRouter({
-    factory: { moduleFactory, integrationFactory, IntegrationHelper },
-    requireLoggedInUser,
-    getUserId: (req) => req.user.getUserId(),
+    factory: {
+        moduleFactory: integrationFactory.moduleFactory,
+        integrationFactory,
+        IntegrationHelper
+    },
+    getUserFromBearerToken,
 });
 
+//todo: what is this route doing here?
```

Comment:
Backend redirect cases, I think we introduced via the Canva use case where we needed to redirect people to our service that we controlled instead of to the frontend of the core application. That happens  lot in apps like Asana, Google workspace, etc. 

Our more recent solutions have been to do that redirect handling in the integration defined router, which didn't exist as a concept at the time of building this thing out.